packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -257,7 +257,7 @@ export const components: ThemeOptions["components"] = {
             borderColor: "transparent",
           },
           "&:disabled": {
-            backgroundColor: "rgba(235, 235, 237, 0.6)",
+            backgroundColor: "transparent",
             color: theme.palette.text.secondary,
```

Comment:
I think the color opacity is not right. 
the default is `rgba(0, 0, 0, 0.26);`, what we have now is `rgba(0, 0, 0, 0.6)`. so the text with the disabled state is still too dark compared to the enabled one.

ref:
![Screenshot 2023-03-28 at 8 58 14 PM](https://user-images.githubusercontent.com/95650076/228399640-c24bfca6-f0f3-41ae-8176-6f3529cd401b.png)

Reply:
@ChrisFu-okta looks are you inspecting from an outdated version of Odyssey? We don't use opacity shifts in `color`; the values come straight from the `text.primary` and `text.secondary` tokens.

<img width="233" alt="Screenshot 2023-03-29 at 9 59 27 AM" src="https://user-images.githubusercontent.com/36284167/228613526-03a18b6a-7b00-4eb2-94af-5b0fc0b0ca80.png">
<img width="120" alt="Screenshot 2023-03-29 at 9 59 07 AM" src="https://user-images.githubusercontent.com/36284167/228613532-7e1b8e7d-db00-4f2a-9587-3296e62f1e40.png">
<img width="121" alt="Screenshot 2023-03-29 at 9 59 02 AM" src="https://user-images.githubusercontent.com/36284167/228613535-cf7b07ea-53f4-4bd2-aad1-086c2201186a.png">

Reply:
sounds good to me. the opacity comes with the default MUI styling.

Reply:
Ah, that's why. We don't export an `IconButton` replacement. Currently, we rec using `Button` with one of our variants instead.